src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -553,8 +553,15 @@ private void AttributePrefix(IEnumerable<HtmlSymbol> whitespace, IEnumerable<Htm
             }
             else
             {
+                // Output the attribute name, the equals and optional quote. Ex: foo="
+                Output(SpanKind.Markup);
+
                 // Not a "conditional" attribute, so just read the value
                 SkipToAndParseCode(sym => IsEndOfAttributeValue(quote, sym));
+
+                // Output the attribute value (will include everything inbetween the attribugte's quotes).
+                Output(SpanKind.Markup);
```

Comment:
And why is this necessary?  Is the important thing splitting the parts up versus ensuring `Output()` is called (which already happens a couple of lines below in this block).